sdks/python/apache_beam/io/__init__.py

```diff
@@ -36,6 +36,7 @@
   from apache_beam.io.gcp.bigquery import *
   from apache_beam.io.gcp.pubsub import *
   from apache_beam.io.gcp import gcsio
+  from apache_beam.transforms._external_transforms.io import *
```

Comment:
One could automate this by having a file in _external_transforms that injects these into the right modules. But this is less magical and seems a nice balance (it only needs to be done per package).